### PR TITLE
Update FTP (un)usage recommendations

### PR DESCRIPTION
**Pull request by meatballhat**

### CODE REVIEW

File: user/deployment/custom.md [hunk 1]
```diff
@@ -9,22 +9,6 @@ You can easily deploy to your own server the way you would deploy from your loca
 You may choose the [Script provider](/user/deployment/script/) instead, as it provides
 easier flexibility with conditional deployment.
 
-### FTP
```
Comment (BanzaiMan) [inline user/deployment/custom.md]: Perhaps we should direct users to the new norm: "Do not use FTP".

Comment (meatballhat) [reply user/deployment/custom.md]: @BanzaiMan I'm going to back out that deletion and provide an SFTP example instead, I think :+1:

Comment (BanzaiMan) [reply user/deployment/custom.md]: That sounds better!

File: user/deployment/custom.md [hunk 2]
```diff
@@ -4,26 +4,35 @@ layout: en
 
 ---
 
-You can easily deploy to your own server the way you would deploy from your local machine by adding a custom [`after_success`](/user/customizing-the-build/) step.
+You can easily deploy to your own server the way you would deploy from your
```
Comment (acnagy) [inline user/deployment/custom.md]: > You can deploy...

(I think we should drop "easily")

File: user/deployment/custom.md [hunk 3]
```diff
@@ -4,26 +4,35 @@ layout: en
...
+local machine by adding a custom [`after_success`](/user/customizing-the-build/)
+step.
 
-You may choose the [Script provider](/user/deployment/script/) instead, as it provides
-easier flexibility with conditional deployment.
+You may choose the [Script provider](/user/deployment/script/) instead, as it
+provides easier flexibility with conditional deployment.
```
Comment (acnagy) [inline user/deployment/custom.md]: > provides easier flexibility with conditional deployment
to
> provides more flexibility with conditional deployment

**Pull request merged**